src-ts/lib/loading-spinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * LoadingSpinner
+ *
+ * Centered Loading Spinner with back overlay
+ */
+import cn from 'classnames'
```

Comment:
Let's change `cn` to `classnames`. Shortening it doesn't buy us anything and is less clear than using a longer, more explicit name.

Reply:
I fixed it.